cypress/e2e/events.cy.ts

```diff
@@ -1,97 +1,306 @@
+    });
+
+    deleteEventSeries(event);
+  });
+
+  it("Series with many instances", () => {
+    const event = {...eventBase};
+    event.title = Math.random().toString(36).slice(2, 7);
+    event.recurType = "weekly_recurring_date";
+
+    createEventSeries(event);
+
+    findEventsInAPI(event).then((events) => {
+      expect(events.length).to.greaterThan(1, `Expected to find multiple events in series, but found ${events.length}`);
+      return events[0];
+    });
+
+    // Let's edit some details, that we can later look up in the API.
+    visitEventEditLink();
+
+    const newTitle = Math.random().toString(36).slice(2, 7);
+    cy.findByLabelText("Title").type(newTitle);
+    cy.findByRole("button", { name: "Save" }).click();
+
+    const newEvent = {...event};
+    newEvent.title = newTitle;
+
+    findEventsInAPI(newEvent).then((events) => {
+      expect(events.length).to.eq(1, `Expected exactly one eventinstance after editing, but found ${events.length}`);
+    });
+
+    // Unpublishing the same instance.
+    cy.get('.breadcrumb').contains(event.title).click();
+    visitEventEditLink();
+
+    cy.get('[id="edit-status-value"]').click();
+    cy.findByRole("button", { name: "Save" }).click();
+
+    findEventsInAPI(newEvent).then((events) => {
+      expect(events.length).to.eq(0, `Expected unpublished event to not show up in API`);
+    });
+
+    deleteEventSeries(event);
   });
 
-  before(() => {
-    cy.drupalLogin("/admin/content/eventseries");
-    // Delete all preexisting instances of each event.
-    cy.get("a")
-      .contains(events.singleEvent.title)
-      .if()
-      .each(() => {
-        // We have to repeat the selector as Cypress will otherwise complain about
-        // missing references to elements when clicking the page.
-        cy.findAllByRole("link", { name: events.singleEvent.title })
-          .first()
-          .click();
-        cy.findByRole("link", {
-          name: `Edit ${events.singleEvent.title}`,
-        }).click();
-        cy.findByRole("button", { name: "More actions" })
-          .click()
-          .parent()
-          .findByRole("link", { name: "Delete" })
-          .click();
-        cy.findByRole("dialog")
-          .findByRole("button", { name: "Delete" })
-          .click();
-
-        // Return to the event list to prepare for the next iteration.
-        cy.visit("/admin/content/eventseries");
+  it("Updating an event using API", () => {
+    const event = {...eventBase};
+    event.title = Math.random().toString(36).slice(2, 7);
+    createEventSeries(event);
+
+    const patchBody = {
+      state: 'SoldOut',
+      external_data: {
+        url: "https://event.local",
+        admin_url: "https://admin.local",
+      }
+    }
+
+    findEventsInAPI(event).then((events) => {
+      const apiEvent = events[0];
+      // Resetting the logged-in session, for calling API.
+      cy.clearCookies();
+      cy.clearAllSessionStorage();
```

Comment:
You can set the state of Cypress to be an anonymous user using [`cy.anonymousUser()`](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/d2c5ea26f8db0b2fa1ab3469a30ef97c5046621e/cypress/support/commands.ts#L98-L98).